components/profile/tabs/information.js

```diff
@@ -0,0 +1,279 @@
+import React, { useEffect, useState } from 'react'
+import { 
+    Button,
+    Container,
+    Dialog,
+    DialogTitle,
+    DialogContent,
+    DialogActions,
+    Grid,
+    Paper,
+    MenuItem,
+    TextField,
+    Select,
+    Skeleton,
+    styled,
+} from '@mui/material';
+import { Icon } from '@iconify/react';
+import axiosClient from '../../../axiosClient';
+
+const genderValues = ['Male', 'Female', 'Unknown'];
+const userTypeValues = ['Basic', 'Moderator', 'Adminstrator'];
+
+const Label = styled('div')(({ theme }) => ({
+    ...theme.typography.h6,
+    padding: theme.spacing(1),
+    textAlign: 'left',
+    color: theme.palette.text.primary,
+}));
+
+const Value = styled('div')(({ theme }) => ({
+    ...theme.typography.body2,
+    padding: theme.spacing(1),
+    textAlign: 'left',
+    color: theme.palette.text.secondary,
+}));
+
+const Config = styled('div')(({ theme }) => ({
+    ...theme.typography.body2,
+    padding: theme.spacing(1),
+    position: 'relative',
+    color: theme.palette.text.primary,
+}));
+
+const ConfigButton = styled(Button)(({ theme }) => ({
+    position: 'absolute',
+    right: '15px',
+}));

```

Comment:
We dont use theme here, you could write without theme for better readability
```suggestion
const ConfigButton = styled(Button)({
    position: 'absolute',
    right: '15px',
})

```